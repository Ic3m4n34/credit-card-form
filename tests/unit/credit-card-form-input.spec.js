import { mount } from '@vue/test-utils';
import CreditCardFormInput from '@/components/credit-card-form-input';

describe('CreditCardFormInput', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = mount(CreditCardFormInput);
  });

  describe('rendering', () => {
    it('renders the component', () => {
      expect(wrapper).toMatchSnapshot();
    });

    it.todo('has a label', () => {

    });

    it.todo('has an input field', () => {

    });
    it.todo('input highlighted when focused', () => {});
  });

  describe('validation', () => {
    it.todo('', () => {});
  });
});
