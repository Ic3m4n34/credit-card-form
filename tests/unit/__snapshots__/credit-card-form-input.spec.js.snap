// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreditCardFormInput rendering renders the component 1`] = `<div class="credit-card-form-input"></div>`;
